src/Microsoft.WSMan.Management/CredSSP.cs

```diff
@@ -440,7 +440,7 @@ public class EnableWSManCredSSPCommand : WSManCredSSPCommandBase, IDisposable/*,
 
         //private const string DelegateComputerParam = "DelegateComputer";
         //private String[] delegatecomputer;
-        //private RuntimeDefinedParameterDictionary dyanmicParameters = new RuntimeDefinedParameterDictionary();
+        //private RuntimeDefinedParameterDictionary dynamicParameters = new RuntimeDefinedParameterDictionary();
```

Comment:
I don't understand how this works... Are these `//` marks not actually comments? Or is this file not compiled?